tests/cross_functional/ui/fusion_access/test_virtualmachine_lifecycle.py

```
@@ -45,6 +60,93 @@ def setup_ui(self, setup_ui_class_factory):
+        child.sendline(vm_username)
+
+        child.expect(r"[Pp]assword:", timeout=60)
+        logger.info("Password prompt detected — sending password")
+        child.sendline(vm_password)
+
+        child.expect(r"\]\$\s", timeout=60)
+        logger.info("Shell prompt detected — logged in successfully")
+
+        return child
+
+    def _calculate_vm_file_md5sum(self, child, test_file, test_data):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_calculate_vm_file_md5sum`

(ANN202)